evalml/tests/component_tests/test_imputer.py

```diff
@@ -122,29 +122,31 @@ def test_categorical_only_input(imputer_test_data):
             "all nan cat",
         ]
     ]
-    y = pd.Series([0, 0, 1, 0, 1])
-    imputer = Imputer()
-    imputer.fit(X, y)
-    transformed = imputer.transform(X, y)
+    y = pd.Series([0, 0, 1, 0, 1] * 4)
+
     expected = pd.DataFrame(
         {
             "categorical col": pd.Series(
-                ["zero", "one", "two", "zero", "three"], dtype="category"
+                ["zero", "one", "two", "zero", "two"] * 4, dtype="category"
             ),
-            "object col": pd.Series(["b", "b", "a", "c", "d"], dtype="category"),
-            "bool col": [True, False, False, True, True],
+            "object col": pd.Series(["b", "b", "a", "c", "d"] * 4, dtype="category"),
+            "bool col": [True, False, False, True, True] * 4,
             "categorical with nan": pd.Series(
-                ["0", "1", "0", "0", "3"], dtype="category"
+                ["0", "1", "0", "0", "3"] * 4, dtype="category"
```

Comment:
FYI, if you are setting the `dtype` of the column explicitly to `category`, Woodwork should recognize this as `Categorical`, regardless of how many values are present.

An example:

```python
>>> import pandas as pd
>>> import woodwork as ww
>>> df = pd.DataFrame({'id': [0], 'vals': pd.Series(["1"], dtype="category")}) 
>>> df.ww.init()
>>> df.ww
       Physical Type Logical Type Semantic Tag(s)
Column                                           
id             int64      Integer     ['numeric']
vals        category  Categorical    ['category']
```

So, that's another option instead of changing series length or updating the inference threshold.